mmedit/datasets/pipelines/augmentation.py

```diff
@@ -50,6 +50,12 @@ class Resize:
         interpolation (str): Algorithm used for interpolation:
             "nearest" | "bilinear" | "bicubic" | "area" | "lanczos".
             Default: "bilinear".
+        backend (str | None): The image resize backend type. Options are `cv2`,
+            `pillow`, `None`. If backend is None, the global imread_backend
+            specified by ``mmcv.use_backend()`` will be used.
+            Default: "pillow".
```

Comment:
Why choose pillow?

Reply:
Sorry, it is copied from another function, default: 'cv2'